src/store/reducers/index.js

```diff
@@ -108,6 +108,10 @@ export const reducer = (
 			blocks.splice( index, 1 );
 			return { blocks: blocks, refresh: ! state.refresh };
 		}
+		case ActionTypes.BLOCK.PARSE: {
+			const parsed = action.parse(action.payload)
+			return { blocks: parsed, refresh: state.refresh };
```

Comment:
Why not import `parse` in this file instead of passing it down every time you want to use it?

Reply:
The idea was to avoid a hard coded dependency.
But for simplicity I'd agree that it would be better to import it directly. 👍